pkg/olog/olog_test.go

```diff
@@ -113,3 +115,35 @@ func TestLogWithAppInfoHook(t *testing.T) {
 		t.Fatalf("unexpected log output (-want +got):\n%s", diff)
 	}
 }
+
+func TestOutputLog(t *testing.T) {
+	// Force JSON handler for valid unmarshaling used in the TestCapturer
+	SetDefaultHandler(JSONHandler)
+
+	logCapture := NewTestCapturer(t)
+	// Initialize test app info
+	app.SetName("ologHooksTest")
+
+	f, err := os.CreateTemp(".", "test_logfile.log")
+	if err != nil {
+		panic(err)
```

Comment:
suggestion: avoid panics
```suggestion
		t.Fatalf("Could not create temp file: %v", err)
```